app/components/arclight/search_result_breadcrumbs_component.html.erb

```diff
@@ -0,0 +1,7 @@
+
+        <% rendered_breadcrumbs = capture { render breadcrumbs } %>
+        <% if rendered_breadcrumbs.present? %>
```

Comment:
Can we pull this conditional up to the `Arclight::SearchResultBreadcrumbsComponent#render?` `Arclight::BreadcrumbComponent` might need one too.